src/components/links/links.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import styled from 'styled-components';
-import { GatsbyLinkProps, Link as GatsbyLink } from 'gatsby';
+import { Link as GatsbyLink } from 'gatsby-plugin-react-i18next';
```

Comment:
What is this Link doing? Aren't all links already "hardcoded" when they reach this `Link` component?